src/WormholeRouter.sol

```diff
@@ -120,14 +113,10 @@ contract WormholeRouter {
      * @param maxFee The maximum amount of fees to pay for the minting of DAI
      */
     function requestMint(WormholeGUID calldata wormholeGUID, uint256 maxFee) external {
-        require(msg.sender == bridges[wormholeGUID.sourceDomain], "WormholeRouter/sender-not-bridge");
-        if(wormholeGUID.targetDomain == l1Domain) {
-            wormholeJoin.registerWormholeAndWithdraw(wormholeGUID, maxFee);
-        } else {
-            address targetBridge = bridges[wormholeGUID.targetDomain];
-            require(targetBridge != address(0), "WormholeRouter/unsupported-target-domain");
-            L1BridgeLike(targetBridge).initiateRequestMint(wormholeGUID, maxFee);
-        }
+        require(msg.sender == targets[wormholeGUID.sourceDomain], "WormholeRouter/sender-not-bridge");
```

Comment:
Let's lose the bridge terminology. Error can be `WormholeRouter/unsupported-source-domain`.